lib/takeaway.rb

```diff
@@ -0,0 +1,94 @@
+require_relative 'restaurant.rb'
+require_relative 'order.rb'
+require_relative 'twilio_messager'
+require 'dotenv'
+
+class Takeaway
+  
+  attr_reader :restaurant, :order_log, :messager
+  
+  def initialize(restaurant = Restaurant.new, order=Order.new, messager=Messager.new) 
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)